cosmic-usage-ui/src/main/resources/public/app/main.js

```diff
@@ -72,6 +147,17 @@ const app = {
     generateReportButtonOnClick: function(event) {
         event.preventDefault();
 
+        var currentSorting = this.getCurrentSorting();
+
+        this.generateSortableReport(currentSorting[0], currentSorting[1]);
```

Comment:
Turning `currentSorting` into an object you could read its values with something like `currentSorting.sortBy` and `currentSorting.sortOrder`.